samples/sample_multi_transcode/src/sample_multi_transcode.cpp

```diff
@@ -341,7 +342,7 @@ void Launcher::Run()
     m_StartTime = GetTick();
 
     // Robust flag is applied to every seession if enabled in one
-    if (m_pSessionArray[0]->pPipeline->GetRobustFlag())
+    if (m_pThreadContextArray[0]->pPipeline->GetRobustFlag())
```

Comment:
I believe here should be actually a result of std::any_of, but this should be another change